prisma/schema.prisma

```diff
@@ -59,13 +47,23 @@ model SpokenLanguage {
 
 // Represents the area a professional works in, such as Gynecology, gastroenterology, oncology, physical therapy, etc.
 model Specialty {
-  id     Int    @id @default(autoincrement())
-  nameJa String
-  nameEn String
+  id    Int             @id @default(autoincrement())
+  names SpecialtyName[]
 
   HealthcareProfessionalSpecialty HealthcareProfessionalSpecialty[]
 }
 
+model SpecialtyName {
```

Comment:
can we use `LocaleName` instead of SpecialtyName? Or does it need distinct fields that are different?

Reply:
oh the middle name and last is a good point. I'm sold 💰